tests/e2e.rs

```diff
@@ -68,9 +69,9 @@ async fn w3c_vc_did_client_direct_post() {
             "definition_id": "0b4dd017-efa6-4a05-a269-9790fa3c22c2",
             "descriptor_map": [
                 {
-                    "id": "vc",
+                    "id": "064255a8-a0fa-4108-9ded-429f83003350",
                     "path": "$",
-                    "format": "jwt_vc_json"
+                    "format": "jwt_vp"
```

Comment:
This is technically not correct, currently the submission is for a `jwt_vc_json`.

Reply:
This will be fixed in #8.